tools/lib/provision.py

```diff
@@ -103,8 +103,11 @@
 if not os.path.exists("/usr/bin/lsb_release"):
     subprocess.check_call(["sudo", "apt-get", "install", "-y", "lsb-release"])
 
-vendor = subprocess_text_output(["lsb_release", "-is"])
-codename = subprocess_text_output(["lsb_release", "-cs"])
+import lsb_release
+
+distro_info = lsb_release.get_distro_information()
+vendor = distro_info['ID']
+codename = distro_info['CODENAME']
```

Comment:
So, the main problem here is that this fails inside the Zulip virtualenv.  The normal fix would be to include `lsb_release` in our `requirements/common.in` file ... except that I can't find an `lsb_release` package on pypi.  Is it possible this is only distributed via `apt` and `yum`?

(the mypy thing is easy to fix; just need an entry in `mypy.ini` for it)

Reply:
OK, we can try the other approach I suggested; if `/etc/lsb_release` exists, parse that, and fall back to the command.  Let's move the little block to do this in `scripts/lib/zulip_tools.py`; then we can import it from both places that need it.